documentation/api/query/v4/aggregate-event-counts.markdown

```diff
@@ -71,6 +71,24 @@ event-count results that were aggregated.
         "total": 3
     } ]
 
+#### Puppet Enterprise
+
+In PE, `successes` and `noops` counts are further subdivided into intentional and corrective parts. Events are mapped to the corresponding
+counts based on value of `corrective_change` flag.
```

Comment:
"the `successes` and `noops` counts" ... "based on the value of `corrective_change`"